.github/workflows/builds.yml

```diff
@@ -235,7 +235,6 @@ jobs:
                 libtool \
                 lz4-devel \
                 make \
-                nss-devel \
```

Comment:
I think libnspr4-dev can be removed with nss where it exists

Reply:
Will do.